packages/components-native/src/AtlantisContext/AtlantisContext.test.tsx

```diff
@@ -19,6 +19,7 @@ const providerValues: AtlantisContextProps = {
     return;
   },
   floatSeparators: { decimal: ".", group: "," },
+  currencySymbol: "€",
```

Comment:
Any specific reason to use this symbol?

Reply:
To test that it is actually using provided values and not just using the default